File: src/Client/UI/MenuAlignment.ts
/**
 * Defines the alignment of a dropdown menu.
 */
export const MenuAlignment = Object.freeze({

	/**
	 * The dropdown menu is left aligned.
	 */
	Start: "Start",

	/**
	 * The dropdown menu is right aligned.
	 */
	End: "End"
});

/**
 * Defines the alignment of a dropdown menu.
 */
export type MenuAlignment = typeof MenuAlignment[keyof typeof MenuAlignment];

File: src/Server/UI/MenuAlignment.cs
namespace Belin.Base.UI;

using System.Text.Json.Serialization;

/// <summary>
/// Defines the alignment of a dropdown menu.
/// </summary>
[JsonConverter(typeof(JsonStringEnumConverter))]
public enum MenuAlignment {

	/// <summary>
	/// The dropdown menu is left aligned.
	/// </summary>
	Start,

	/// <summary>
	/// The dropdown menu is right aligned.
	/// </summary>
	End
}
